src/+FeatureCreators/LCDFeatureSet.m

```diff
@@ -0,0 +1,202 @@
+classdef LCDFeatureSet < FeatureCreators.Base
+% Largest Common Denominator feature set Specifies a feature set consisting of:
+%   see LCDFeatureSet.getAFErequests()
+
+    %% --------------------------------------------------------------------
+    properties (SetAccess = private)
+        deltasLevels;
+    end
+    
+    %% --------------------------------------------------------------------
+    methods (Static)
+    end
+    
+    %% --------------------------------------------------------------------
+    methods (Access = public)
+        
+        function obj = LCDFeatureSet( )
+            obj = obj@FeatureCreators.Base();
+        end
+        %% ----------------------------------------------------------------
+
+        function afeRequests = getAFErequests( obj )
+            commonParams = {...
+                'pp_bNormalizeRMS', true, ...
+                'pp_intTimeSecRMS', 4, ...
+                'pp_bBinauralRMS', false, ...
+                'fb_type', 'gammatone', ...
+                'fb_lowFreqHz', 80, ...
+                'fb_highFreqHz', 8000, ...
+                'ihc_method', 'halfwave', ... % stream segr. uses 'dau'
+                'ild_wSizeSec', 20E-3, ...  % DnnLoc uses 20E-3, stream segr. uses 25E-3
+                'ild_hSizeSec', 10E-3, ...
+                'rm_wSizeSec', 20E-3, ... % DnnLoc uses 20E-3, identification uses 25E-3
+                'rm_hSizeSec', 10E-3, ... % DO NOT CHANGE -- important for gabor filters
+                'rm_scaling', 'power', ... % DnnLoc uses power, identification uses magnitude
+                'rm_decaySec', 8E-3, ...
+                'cc_wSizeSec', 20E-3, ... % dnnLocKs uses 20E-3, stream segr. uses 25E-3
+                'cc_hSizeSec', 10E-3, ... % dnnLocKs uses 10E-3, stream segr. uses 10E-2
+                'cc_wname', 'hann' ...,
+                'cc_maxDelaySec', 0.001,...
+                };
+            afeRequests{1}.name = 'amsFeatures';
+            afeRequests{1}.params = genParStruct( ...
+                commonParams{:}, ...
+                'fb_nChannels', 16, ...
+                'ams_fbType', 'log', ...
+                'ams_nFilters', 8, ...
+                'ams_lowFreqHz', 2, ...
+                'ams_highFreqHz', 256', ...
+                'ams_wSizeSec', 128e-3, ...
+                'ams_hSizeSec', 32e-3 ...
+                );
+            afeRequests{2}.name = 'spectralFeatures';
+            afeRequests{2}.params = genParStruct( ...
+                commonParams{:}, ...
+                'fb_nChannels', 32 ...
+                );
+            afeRequests{3}.name = 'ratemap';
+            afeRequests{3}.params = genParStruct( ...
+                commonParams{:}, ...
+                'fb_nChannels', 32 ...
+                );
+            afeRequests{4}.name = 'gabor';
+            afeRequests{4}.params = genParStruct( ...
+                commonParams{:}, ...
+                'fb_nChannels', 24 ...

```

Comment:
Can we make `fb_nChannels` also a common parameter? Most of requests use 32, not this `Gabor` feature.